### ISSUE TITLE
Console flooded with too many debug messages

### ISSUE THREAD

**Issue by mvelbaum** [openziti/ziti-browzer-bootstrapper#205]
All the `tls_*()` function have `console.log()` calls which end up massively spamming the web console, especially on services that continuously send traffic.

**Comment by rentallect** [comment]
Agreed.  This is some debugging output from my wasm.  I'll clean that up soon.

**Comment by qrkourier** [comment]
The default console log level for ZBR now seems to be "silent."
